dependencies.yaml

```diff
@@ -250,18 +256,6 @@ dependencies:
           - numpy>=1.23.4,<3.0a0
           - scikit-image>=0.19.0,<0.26.0a0
           - scipy>=1.11.2
-      - output_types: conda
-        packages:
-          - cupy>=12.0.0
-          # Not sure where these go, if anywhere:
-          # - openslide
-          # - xorg-libxcb
-    specific:
-      - output_types: [requirements, pyproject]
-        matrices:
-          - matrix: # All CUDA 12 versions
-            packages:
-              - cupy-cuda12x>=12.0.0
```

Comment:
Previously we just had this logic with CUDA 11 & 12

Could we do something similar for CUDA 13.0?

https://github.com/rapidsai/cucim/blob/becb456415439c263b9c11a0bd11737b26ae60a9/dependencies.yaml#L289-L315